rfc8446.xml

```diff
@@ -826,7 +826,7 @@ at runtime is not prescribed by the presentation language.</t>
 
 <t>The handshake protocol is used to negotiate the security parameters
 of a connection. Handshake messages are supplied to the TLS record layer, where
-they are encapsulated within one or more TLSPlaintext or TLSCiphertext structures, which are
+they are encapsulated within one or more TLSPlaintext or TLSCiphertext structure  which are
```

Comment:
I think there's a rule for whether to use singular or plural in these cases, but  I forget what the rule says